### ISSUE TITLE
Feature of this provider

### ISSUE THREAD

**Issue by FAMEEXE** [GibraltarSoftware/VistaDB.EFCore#14]
Hi, I'm waiting for a EF Core provider for VistaDB, but this project does not seem to be developed anymore...
Are there any plans to release this provider in the future, or is this project not maintained anymore?

**Comment by kendallmiller** [comment]
TLDR; We've put this on hold while we wait for a specialist resource for EF Core provider development to be available.  We intend to pick it back up likely later this year.

We recognize that EF Core support is essential for modern .NET and are working through the problems to make a provider.  Here's some insight into the challenges we're facing:

It's been very difficult to port the SqlServer provider to work with VistaDB largely because there just isn't documentation for how to make an EF provider that's been updated after .NET Core 3.. and a lot has changed since then.  There are only a few providers that have been written, and while the SqlServer one is open source the implementation approach is extraordinarily complex.  So, we've had difficulty separating out what applies and doesn't apply to VistaDB.

This challenge is largely driven by the approach the EF provider test path takes - namely that it relies on migrations & code first to work.  This is a challenge for us because while VistaDB supports the T-SQL syntax of SQL Server, there are significant differences when it comes to how you do schema queries and changes (these are naturally very engine specific).  So, we never supported migrations or code first schema creation in EF 6 or prior versions of EF.   That means we don't have a template to work from and are trying to work out an approach "from scratch".
